scripts/sbc-kill-broker.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
Not sure what the total sum of SBC related scripts will be, but WDYT about creating a distinct folder `scripts/sbc/` and have these scripts live there?

Reply:
Agreed - if we retain the scripts I will move them to `scripts/sbc` ... now considering though whether we need them at all, or just some documented demo steps/commands.

Reply:
@javabrett IMO the scripts are helpful to have for validation.  For example, pre-GA as the CP releases are rev'd, I use [validation scripts](https://github.com/confluentinc/cp-demo/tree/6.1.0-post/scripts/validate) as a sanity test to verify everything is working.  It's a poor man's version of what a full testing workflow should be, but it's pretty effective to make sure the more complicated pieces of cp-demo work...beyond just a successful `scripts/start.sh` run.

So maybe we can think about docs decoupled from validation: what you decide to put into docs for SBC may look different than what you decide to put into a validation script for SBC.  You could omit the script, but I'd love to have _something_ if possible as a quick sanity check.